File: BAEKJOON/11650.py
n=int(input())
array=[]
for i in range(n):
    input_data=input().split()
    array.append((int(input_data[0]),int(input_data[1])))

result=sorted(array,key=lambda x:(x[0],x[1]))
for i in range(len(result)):
    print(result[i][0], result[i][1])

File: BAEKJOON/sort/11650.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().split("\n");

const n = Number(input[0]);
const arr = [];
for (let i = 1; i <= n; i++) {
  let [x, y] = input[i].split(" ").map(Number);
  arr.push([x, y]);
}
arr.sort(compare);
let result = "";
for (let [x, y] of arr) {
  result += x + " " + y + "\n";
}
console.log(result);

function compare(a, b) {
  if (a[0] != b[0]) return a[0] - b[0];
  else return a[1] - b[1];
}
